go.mod

```diff
@@ -1,18 +1,24 @@
 module github.com/liatrio/liatrio-otel-collector
 
-go 1.21
+go 1.22.2
 
-require github.com/liatrio/liatrio-otel-collector/receiver/gitproviderreceiver v0.0.0-00010101000000-000000000000
+require (
+	github.com/liatrio/liatrio-otel-collector/extension/githubappauthextension v0.0.0-00010101000000-000000000000
+	github.com/liatrio/liatrio-otel-collector/receiver/gitproviderreceiver v0.0.0-00010101000000-000000000000
+)
 
 require (
 	github.com/Khan/genqlient v0.7.0 // indirect
+	github.com/bradleyfalzon/ghinstallation/v2 v2.10.0 // indirect
 	github.com/felixge/httpsnoop v1.0.4 // indirect
 	github.com/fsnotify/fsnotify v1.7.0 // indirect
 	github.com/go-logr/logr v1.4.1 // indirect
 	github.com/go-logr/stdr v1.2.2 // indirect
 	github.com/go-viper/mapstructure/v2 v2.0.0-alpha.1 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/golang-jwt/jwt/v4 v4.5.0 // indirect
 	github.com/golang/snappy v0.0.4 // indirect
+	github.com/google/go-github/v60 v60.0.0 // indirect
```

Comment:
`v60` is here because it's imported by the `ghinstallation` library for GitHub App Auth.